build/configs/rtl8730e/hello_16M/defconfig

```diff
@@ -155,7 +155,7 @@ CONFIG_AMEBASMART_SPI1=y
 CONFIG_AMEBASMART_SPI_EXCHANGE=y
 # CONFIG_AMEBASMART_I2C is not set
 # CONFIG_AMEBASMART_I2S is not set
-
+# CONFIG_AMEBASMART_MIPI is not set
```

Comment:
Let's revert changes on defconfigs.
1. There are conflicts.
2. What you modified does not affect real operation. (It's added with the config off)

So, we don't need to make changes on them.

Reply:
Dear @sunghan-chang,
I have removed them in defconfigs
Thank you

Reply:
Dear @sunghan-chang,
I have removed changes in defconfig